examples/lock/apko.yaml

```diff
@@ -0,0 +1,24 @@
+contents:
+  repositories:
+    - https://dl-cdn.alpinelinux.org/alpine/edge/main
+  packages:
+    - busybox
+
+
+# optional environment configuration
+environment:
+  PATH: /usr/sbin:/sbin:/usr/bin:/bin
+
+cmd: /bin/sh -l
+
+# data for /etc/os-release if it does not already exist
+# in the image
+os-release:
+  id: alpine
+  version-id: '3.16.0'
+  name: 'Alpine Slim'
+  pretty-name: 'Alpine Slim (apko)'
```

Comment:
We can remove this if we add the package `alpine-baselayout` (or for Wolfi, `wolfi-baselayout`)

Reply:
this is an example so it is good that we assert that official alpine packages still work. we can add wolfi as an example later